CMakeLists.txt

```diff
@@ -192,7 +195,7 @@ if(NOT Protobuf_FOUND)
   set(CMAKE_FIND_PACKAGE_PREFER_CONFIG OFF)
 endif()
 
-message(STATUS "Using pbf headers from ${PROTOBUF_INCLUDE_DIR}")
+message(STATUS "Using pbf headers from ${PROTOBUF_INCLUDE_DIRS}")
```

Comment:
this is weird.. I compiled on intel mac and all of a sudden it wouldn't see the protobuf include dir anymore.. I dug a bit and found that the `protobuf-options.cmake` was using the plural version. seems to be working on linux too, probably there both is defined